### PR TITLE
[DOCFIX] Improve Configuration webpage with cluster default

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/en/Configuration-Settings.md [hunk 1]
```diff
@@ -81,9 +81,27 @@ $ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
...
-> these site property files, unless `${ALLUXIO_HOME}/conf` is on the application classpath.
+## Use Cluster Default
+
+Since v1.8, each Alluxio client can initialize the configuration with cluster-wise
+configuration values fetched from masters.
+To be specific, when different client applications like Alluxio Shell commands,
+Spark jobs, or MapReduce jobs connect to an Alluxio service,
+they will initialize their own Alluxio configuration properties with the default values
+supplied by the masters based on the `${ALLUXIO_HOME}/conf/alluxio-site.properties` files on
+masters. As a result, if masters put client-side settings (e.g., `alluxio.user.*`) or
```
Comment (aaudiber) [inline docs/en/Configuration-Settings.md]: The "As a result..." sentence is a little hard to follow since it's so long - could we break to 2 or 3 shorter sentences?

File: docs/en/Configuration-Settings.md [hunk 2]
```diff
@@ -81,9 +81,27 @@ $ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
...
+`${ALLUXIO_HOME}/conf/alluxio-site.properties`,
+these properties will become cluster-wise default values for new Alluxio clients
+unless these clients overwrite the same properties using the approaches described in
+[Configure Alluxio for Applications](Configuration-Settings.html#configure-applications),
+or specify property `alluxio.user.conf.cluster.default.enabled=false` to
+decline the cluster-wise default values.
+
+> Note that, before v1.8, `${ALLUXIO_HOME}/conf/alluxio-site.properties` file is only loaded by
+Alluxio server
+> processes and will be ignored by applications interacting with Alluxio service through Alluxio
```
Comment (aaudiber) [inline docs/en/Configuration-Settings.md]: won't pre-v1.8 clients still use alluxio-site.properties if it's on their classpath?

File: docs/en/Configuration-Settings.md [hunk 3]
```diff
@@ -81,9 +81,27 @@ $ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
 Make sure that this file is distributed to `${ALLUXIO_HOME}/conf` on every Alluxio node (masters
 and workers) before starting the cluster.
 
-> Note that, `${ALLUXIO_HOME}/conf/alluxio-site.properties` are only loaded by Alluxio server
-> processes. Applications interacting with Alluxio service through Alluxio client will not load
-> these site property files, unless `${ALLUXIO_HOME}/conf` is on the application classpath.
+## Use Cluster Default
+
+Since v1.8, each Alluxio client can initialize the configuration with cluster-wise
```
Comment (aaudiber) [inline docs/en/Configuration-Settings.md]: I think the uses of cluster-wise should say cluster-wide

File: docs/en/Configuration-Settings.md [hunk 4]
```diff
@@ -179,6 +197,35 @@ earliest in this list:
 server process including master and worker searches `alluxio-site.properties` in a list paths of
 `${HOME}/.alluxio/`, `/etc/alluxio/` and `${ALLUXIO_HOME}/conf` in order, and will skip the
 remaining paths once this `alluxio-site.properties` file is found.
+4. [Cluster default values](#use-cluster-default). An Alluxio client may initialize its
+configuration based on the configuration fetched from the master nodes as the cluster-wise default.
 
 If no above user-specified configuration is found for a property, Alluxio runtime will fallback to
 its [default property value](Configuration-Properties.html).
+
+To check the value of a specific configuration property and the source of its value, user can use
```
Comment (aaudiber) [inline docs/en/Configuration-Settings.md]: user -> users

File: docs/en/Configuration-Settings.md [hunk 5]
```diff
@@ -179,6 +197,35 @@ earliest in this list:
...
+To list all of the configuration properties with sources:
+
+```bash
+$ bin/alluxio getConf --source
+alluxio.conf.dir=/Users/bob/alluxio/conf (SYSTEM_PROPERTY)
+alluxio.debug=false (DEFAULT)
+...
+```
+
+To list all of the configuration properties by the master with sources (rather than the local
```
Comment (aaudiber) [inline docs/en/Configuration-Settings.md]: configuration properties by the master -> master-side configuration properties

Comment (aaudiber) [inline docs/en/Configuration-Settings.md]: can we mention that this requires the cluster to be running, while the other getConf commands do not?

File: docs/en/Configuration-Settings.md (hunk 3 above)
Comment (apc999) [reply docs/en/Configuration-Settings.md]: oops. fixed

File: docs/en/Configuration-Settings.md (hunk 1 above)
Comment (apc999) [reply docs/en/Configuration-Settings.md]: done

File: docs/en/Configuration-Settings.md (hunk 2 above)
Comment (apc999) [reply docs/en/Configuration-Settings.md]: added.

File: docs/en/Configuration-Settings.md (hunk 4 above)
Comment (apc999) [reply docs/en/Configuration-Settings.md]: done

**Comment by apc999** [comment]
@aaudiber PTAL

**Comment by apc999** [comment]
@gpang with this change, is there anything to change/simplify on the secure HDFS page?

File: docs/en/Configuration-Settings.md [hunk 6]
```diff
@@ -220,8 +223,8 @@ alluxio.debug=false (DEFAULT)
 ...
 ```
 
-To list all of the configuration properties by the master with sources (rather than the local
-settings):
+Users can also specify `--master` option to list all of the  master-side configuration properties.
+Note that, with `--master` option `getCOnf` queries the master and thus require the master nodes running; in contrast, without `--master` option it only checks the local configuration.
```
Comment (aaudiber) [inline docs/en/Configuration-Settings.md]: nit: getCOnf
